WooCommerce/Classes/ViewRelated/ReusableViews/SpacerTableViewCell.xib

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm curious if you've tried not having a XIB. Will it work? It seems unnecessary. 😄 This is nitpicky so feel free to ignore.

Reply:
I thought about not using a xib for 1 subview, though I generally still create a xib whenever possible in Xcode (`UITableViewHeaderFooterView` does not have a xib option in Xcode new file window) following our practices in the codebase. Also, our table view cell registration usually expects the use of a xib (`tableView.registerNib(for:)`) so I thought it's easier not to create exception for table view configuration 😅 .

Xib does create tens of XML diffs even it just has one subview 🙄 looking forward to gradually moving to SwiftUI!